panel/src/components/Writer/Marks/Bold.js

```diff
@@ -1,11 +1,12 @@
 import Mark from "../Mark"
+import Vue from "vue";
```

Comment:
If `window.panel.app.$t()` works, that would be preferred, I think.

Reply:
As a result, which one should we continue with? Can we use `TranslationString` instead `Vue` in this PR?

Reply:
> If window.panel.app.$t() works, that would be preferred, I think.

Works.

Isn't there a shorter way to this job? Like `window.t('key')`

Reply:
Wouldn't we have the same problem that we wouldn't know whether the app is already loaded?

I think we can see the plugin use-case separately from our own code – here we *can* import the class.

Reply:
The problem with translations not being loaded would completely vanish with the switch to Inertia. Another reason to move that forward.

Reply:
Does anything speak against keeping importing Vue for this PR and fix that later, once we have Inertia running? I think it does not make sense to keep this blocked just because of that, right?

Reply:
Let's wait for Inertia - I think that one will solve it quite nicely

Reply:
I think it makes sense then to already merge this PR to avoid merge conflicts with Inertia later.

Reply:
Just replied to the answer above. 

Ok, what about merging it right now and already adding a todo to the inertia PR? We have way too many open PRs, so being able to close a PR feels like a good choice to me :)

Reply:
I think we should add TODO tags then to all the places, otherwise it's too easy to forget about things after Inertia.

Reply:
`@todo` added for all `Vue.$t()` in this PR